clowdapp.yaml

```diff
@@ -0,0 +1,116 @@
+---
+apiVersion: v1
+kind: Template
+metadata:
+  name: ros
+objects:
+- apiVersion: cloud.redhat.com/v1alpha1
+  kind: ClowdApp
+  metadata:
+    name: ros
+  spec:
+    envName: ${ENV_NAME}
+    dependencies:
+    - rbac
+    pods:
+      - name: backend
+        minReplicas: 1
+        image: ${IMAGE}:${IMAGE_TAG}
+        command: ["bash"]
+        args: ["-c", "python -m manage db upgrade && python -m ros.api.main"]
```

Comment:
I'm assuming this is a Django app? If so I've seen other apps break out their db migrations into init containers. If your app is setup to always handle `db upgrade`, then this is fine. Just a thought.